docs/user-guide/slicing.ipynb

```diff
@@ -217,6 +226,174 @@
     "d['x', 1:2]['a'] == d['a']['x', 1:2]\n",
     "d['x', 1:2]['a'].coords['x'] == d.coords['x']['x', 1:2]"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Value based slicing\n",
+    "\n",
+    "Data in a [Dataset](../generated/scipp.Dataset.rst#scipp.Dataset) or [DataArray](../generated/scipp.DataArray.rst#scipp.DataArray) can be sliced by the coordinate value.\n",
+    "Value indexing with a [Variable](../generated/scipp.Variable.rst#scipp.Variable)  or an [Variable](../generated/scipp.Variable.rst#scipp.Variable) range is made via `__getitem__` and `__setitem__` with a dimension label as first argument.\n",
+    "This is available for DataArrays and Datasets, as well as items of a Dataset, but not for Variable types as coordinates are necessary.\n",
+    "In all cases a *view* is returned, i.e., just like when slicing a [numpy.ndarray](https://docs.scipy.org/doc/numpy/reference/generated/numpy.ndarray.html#numpy.ndarray) no copy is performed.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Value slicing on point coordinates\n",
```

Comment:
I don't think scipp is or should adopt this Mantid-termonology "point coordinate". This is used nowhere else in the docs. Can we just avoid naming this here, i.e.,this is the "normal" case, and only the "special" case of bin-edges needs a name?